### DISCUSSION TITLE
Descontos e taxas....

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1582] [Q&A]
Saudações galera!!

Estou usando mapos há 10 dias.que paz......
Hoje me surgiu uma necessidade. Como informar desconto ou alguma taxa na OS e na venda?
Exemplo: Cliente extrapolou a estadia da máquina na loja. Quando isso ocorre, cobro R$100 de multa (NUNCA mais cliente "esqueceu" pc aqui na loja depois disso). Então eu preciso aplicar a taxa e, as vezes, dou desconto DEPENDENDO da história do cliente.
Onde lanço esse desconto? PS: a taxa estou lançando como serviço.

**Comment by Pr3d4dor** [comment]
@bernarcom Como não tem a opção de desconto no sistema, o que o pessoal faz é adicionar um serviço ou produto chamado desconto, e aí vc edita o banco de dados manualmente coloca um valor negativo (por exemplo -1) e com isso você adicionar ele na OS ou na venda para ter desconto.

**Reply by SuperPorte** [reply]
Qual o caminho para fazer essa alteração do banco?

**Reply by Pr3d4dor** [reply]
@bernarcom Na tabela de servicos ou produtos você edita o que tiver o nome "Desconto" e coloca o valor -1. Pelo phpadmin você navega no banco de dados utilizando o menu na esquerda.

**Reply by SuperPorte** [reply]
Consegui editar o campo, mas quando adiciono na OS, ela fica carregando infinitamente e não entra na edição da OS =(

**Reply by guskid** [reply]
> Consegui editar o campo, mas quando adiciono na OS, ela fica carregando infinitamente e não entra na edição da OS =(

Tentei fazer isso aqui e ficou carregando infinitamente tb

**Reply by marllonferreira** [reply]
no canal **MapOS Oficial** tem vários vídeos interessante inclusive esse do desconto por valor negativo

https://youtu.be/rbWGL2lgBcQ

**Reply by SuperPorte** [reply]
Assisti o vídeo agora. Precisa de remover a validação de preço negativo. Relativamente simples.

**Comment by willph** [comment]
> Saudações galera!!
> 
> Estou usando mapos há 10 dias.que paz...... Hoje me surgiu uma necessidade. Como informar desconto ou alguma taxa na OS e na venda? Exemplo: Cliente extrapolou a estadia da máquina na loja. Quando isso ocorre, cobro R$100 de multa (NUNCA mais cliente "esqueceu" pc aqui na loja depois disso). Então eu preciso aplicar a taxa e, as vezes, dou desconto DEPENDENDO da história do cliente. Onde lanço esse desconto? PS: a taxa estou lançando como serviço.

Foi enviado uma Atualização com relação ao desconto em OS e Vendas
Sobre taxas, ainda não tem esse campo.
